### PR TITLE
Make Tiles and NavPage items <a> in DOM

### PR DESCRIPTION
**Pull request by taysea**
Motivated by Issue #256: In order for links to be crawlable, elements with href need to be <a> in the DOM. This PR adjust the home tiles and navpage items to be <a href=... /> in the DOM.

Resource here: https://support.google.com/webmasters/answer/9112205?hl=en

### CODE REVIEW

**Comment by taysea** [comment]
Will close: #256

File: aries-site/src/layouts/navigation/NavPage.js [hunk 1]
```diff
@@ -16,21 +16,26 @@ const NavItem = ({ item, topic }) => {
...
+        <Box
+          direction="row"
+          margin={{ vertical: 'large' }}
+          gap={size !== 'small' ? 'large' : 'medium'}
+        >
+          {/* Adds placeholder icon for the meantime
+           * while we wait to get all the icons */}
+          {itemData.icon
+            ? itemData.icon('xlarge')
+            : iconsMap.branding('xlarge')}
```
Comment (halocline) [inline aries-site/src/layouts/navigation/NavPage.js]: Is this condition still needed? Opportunity to eliminate while we are in here?

File: aries-site/src/layouts/navigation/NavPage.js [hunk 2]
```diff
@@ -16,21 +16,26 @@ const NavItem = ({ item, topic }) => {
...
-        {/* Adds placeholder icon for the meantime
-         * while we wait to get all the icons */}
-        {itemData.icon ? itemData.icon('xlarge') : iconsMap.branding('xlarge')}
-        <Box>
-          <Text weight="bold" size={size !== 'small' ? 'large' : 'medium'}>
-            {itemData.name}
-          </Text>
-          <Paragraph size="small">{itemData.description}</Paragraph>
+      {/* Needs to be <a> in DOM for web crawling: https://support.google.com/webmasters/answer/9112205?hl=en */}
+      <Button fill>
```
Comment (halocline) [inline aries-site/src/layouts/navigation/NavPage.js]: This approach makes sense to me.

File: aries-site/src/layouts/navigation/NavPage.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/layouts/navigation/NavPage.js]: The only reason I would leave it is if we think there'd be a case where a page didn't have an icon yet and it we want to leave it as a fallback -- at least we'd have confidence that the page layout would look correct as pages get added.

File: aries-site/src/pages/index.js [hunk 3]
```diff
@@ -25,11 +25,15 @@ const HomeTiles = ({ ...rest }) => {
...
   return (
+    // Needs to be <a> in DOM for web crawling: https://support.google.com/webmasters/answer/9112205?hl=en
     <Tile
+      as="a"
       pad="medium"
       background={topic.color}
       key={topic.color}
       ref={ref}
+      fill
+      style={{ textDecoration: 'none' }}
```
Comment (ShimiSun) [inline aries-site/src/pages/index.js]: I'd consider creating a new styled component of <AnchoredTile /> that will have a combination of lines 30, 35 and 36, to distinguish it from a regular tile, not a must but cleaner and more readable.

File: aries-site/src/layouts/navigation/NavPage.js (hunk 2 above)
Comment (halocline) [reply aries-site/src/layouts/navigation/NavPage.js]: I believe the `passHref` prop from line 18 is getting passed to the button, which then is being treated as `<Button>` with an `href` prop.

Comment (taysea) [reply aries-site/src/layouts/navigation/NavPage.js]: Yes, what Matt said is correct, the passHref puts the href on the button and then Grommet knows to make it an <a> in the DOM.

File: aries-site/src/pages/index.js (hunk 3 above)
Comment (taysea) [reply aries-site/src/pages/index.js]: Pulling over my reply on slack:
I'm open to that, I didn't want to refactor to another component if it didn't seem necessary, but if I were to, I could call it `AnchorTile`?

Comment (ShimiSun) [reply aries-site/src/pages/index.js]: `AnchorTile` sounds good to me.

File: aries-site/src/layouts/navigation/NavPage.js (hunk 2 above)
Comment (ShimiSun) [reply aries-site/src/layouts/navigation/NavPage.js]: Correct, I was looking for the `href` on the Button level.  thanks for clarifying.

**Pull request merged**